airbyte-config/config-persistence/src/test/java/io/airbyte/config/persistence/WorkspaceFilterTest.java

```diff
@@ -0,0 +1,126 @@
+            .execute());
+    // create actors
+    database.transaction(ctx ->
+        ctx.insertInto(ACTOR, ACTOR.WORKSPACE_ID, ACTOR.ID, ACTOR.ACTOR_DEFINITION_ID, ACTOR.NAME, ACTOR.CONFIGURATION, ACTOR.ACTOR_TYPE)
+            .values(WORKSPACE_ID_0, ACTOR_ID_0, SRC_DEF_ID, "ACTOR-0", JSONB.valueOf("{}"), ActorType.source)
+            .values(WORKSPACE_ID_1, ACTOR_ID_1, SRC_DEF_ID, "ACTOR-1", JSONB.valueOf("{}"), ActorType.source)
+            .values(WORKSPACE_ID_2, ACTOR_ID_2, DST_DEF_ID, "ACTOR-2", JSONB.valueOf("{}"), ActorType.source)
+            .values(WORKSPACE_ID_3, ACTOR_ID_3, DST_DEF_ID, "ACTOR-3", JSONB.valueOf("{}"), ActorType.source)
+            .execute());
+    // create connections
+    database.transaction(ctx ->
+        ctx.insertInto(CONNECTION, CONNECTION.SOURCE_ID, CONNECTION.DESTINATION_ID, CONNECTION.ID, CONNECTION.NAMESPACE_DEFINITION, CONNECTION.NAME,
+                CONNECTION.CATALOG, CONNECTION.MANUAL)
+            .values(ACTOR_ID_0, ACTOR_ID_1, CONN_ID_0, NamespaceDefinitionType.source, "CONN-0", JSONB.valueOf("{}"), true)
+            .values(ACTOR_ID_0, ACTOR_ID_2, CONN_ID_1, NamespaceDefinitionType.source, "CONN-1", JSONB.valueOf("{}"), true)
+            .values(ACTOR_ID_1, ACTOR_ID_2, CONN_ID_2, NamespaceDefinitionType.source, "CONN-2", JSONB.valueOf("{}"), true)
+            .values(ACTOR_ID_1, ACTOR_ID_2, CONN_ID_3, NamespaceDefinitionType.source, "CONN-3", JSONB.valueOf("{}"), true)
+            .values(ACTOR_ID_2, ACTOR_ID_3, CONN_ID_4, NamespaceDefinitionType.source, "CONN-4", JSONB.valueOf("{}"), true)
+            .values(ACTOR_ID_3, ACTOR_ID_1, CONN_ID_5, NamespaceDefinitionType.source, "CONN-5", JSONB.valueOf("{}"), true)
+            .execute());
+    // create jobs
+    final OffsetDateTime currentTs = OffsetDateTime.now();
+    database.transaction(ctx ->
+        ctx.insertInto(JOBS, JOBS.UPDATED_AT, JOBS.SCOPE)
+            .values(currentTs.minusHours(0), CONN_ID_0.toString())
+            .values(currentTs.minusHours(5), CONN_ID_0.toString())
+            .values(currentTs.minusHours(10), CONN_ID_1.toString())
+            .values(currentTs.minusHours(15), CONN_ID_1.toString())
+            .values(currentTs.minusHours(20), CONN_ID_2.toString())
+            .values(currentTs.minusHours(30), CONN_ID_3.toString())
+            .values(currentTs.minusHours(40), CONN_ID_4.toString())
+            .values(currentTs.minusHours(50), CONN_ID_4.toString())
+            .values(currentTs.minusHours(70), CONN_ID_5.toString())
+            .execute());
+  }
+
+
+  @BeforeEach
+  void beforeEach() {
+    setup();
+  }
+
+  void setup() {
+    configRepository = spy(new ConfigRepository(
+        database,
+        new ActorDefinitionMigrator(new ExceptionWrappingDatabase(database)),
+        null));
+  }
+
+
+  @Test
+  @DisplayName("should return the a list of workspace IDs with most recently running jobs")
+  void testListWorkspacesByMostRecentlyRunningJobs() throws IOException {
```

Comment:
Can you add a comment saying how the tests are setup and why eventually those 3 workspaces were selected, so users do not have to read through all setup code when they need a closer look at this test?

Reply:
sure, added some comments below.